src/common/hist_util.cc

```diff
@@ -229,8 +229,7 @@ uint32_t HistCutMatrix::GetBinIdx(const Entry& e) {
 
 void GHistIndexMatrix::Init(DMatrix* p_fmat, int max_num_bins) {
   cut.Init(p_fmat, max_num_bins);
-  const int32_t nthread = omp_get_max_threads();
-  // const int nthread = 1;
+  size_t nthread = omp_get_max_threads();
```

Comment:
Make this `const`?

Reply:
it is being changed later

Reply:
See comment below. Probably safer to declare another variable there.